sopel/modules/tell.py

```diff
@@ -65,7 +65,7 @@ def dumpReminders(fn, data, lock):
     return True
 
 
-def setup(self):
+def setup(self):  # TODO: `self`?!? but why!? change to `bot` eventually...
```

Comment:
Just change it now, tbh. Separate commit if you like.

Basically, either fix it (and don't leave a comment) or remove the comment (because it's unrelated to the PR).

Reply:
Separate commit. Updated this as well:
https://github.com/sopel-irc/sopel/blob/39e3680db18c4bed9801d040f7486c655b95a9a0/sopel/modules/tell.py#L74

```diff
-    except (OSError, IOError):  # Remove IOError when dropping py2 support 
+    except (OSError, IOError):  # TODO: Remove IOError when dropping py2 support 
```